edurange_refactored/templates/notification_history/components/Notification.js

```diff
@@ -0,0 +1,75 @@
+import './Notification.css';
+import { createRoot } from 'react-dom/client';
+import React, { useState, useEffect } from 'react';
+
+function Notification(props) {
+  const notif_json = JSON.parse(props.notifications, (key, value) => {
+    if (typeof value === 'number') {
+      return value.toString();
+    }
+    return value;
+  });
+
+  const notifCount = Object.keys(notif_json).length;
+
+  const myNotifs = [];
+
+  for (let i = 0; i < notifCount; i++) {
+    const notif_id = notif_json[i].id;
+    const notif_timeStamp = notif_json[i].date;
+    const notif_content = notif_json[i].detail;
+    myNotifs.push([notif_id, notif_timeStamp, notif_content]);
+  };
+
+  console.log(myNotifs);
+
+  const buildNotificationTable = (input) => {
+    return input.map((item, index) => (
+      <tr key={index}>
+        <td>{item[0]}</td>
+        <td>{item[1]}</td>
+        <td>{item[2]}</td>
+      </tr>
+    ));
+  };
+
+  const notifTemp = buildNotificationTable(myNotifs);
+
+  return (
+    <div id="notification-page">
```

Comment:
Not sure if it's because of trouble using the bootstrap "container" class, but I'm not a fan of how this div touches the sides of the window, some margins would be nice